openbb_platform/extensions/equity/openbb_equity/fundamental/fundamental_router.py

```diff
@@ -93,18 +82,18 @@ def divs(
 
 
 @router.command(model="EarningsCalendar")
-def earning(
+def earnings(
```

Comment:
Whats different here and the earnings calendar above?

Reply:
This is not actually an earnings calendar.  This function actually returns,`historical_eps`

Reply:
still need a decision on this one. There is a bug on the models side where the model does not return what it should and yes there is a separate calendar `obb.equity.calendar.earnings` command

Reply:
IIUC this question will be addressed in this PR
https://github.com/OpenBB-finance/OpenBBTerminal/pull/5690

That PR will also need to address the renaming of the historical earnings model